### ISSUE TITLE
Add Video/Audio chat to Neko

### ISSUE THREAD

**Issue by gbrian** [helppery-com/meetnav-frontend-base#11]
# Help wanted payable
***Please read [how to collaborate](https://github.com/helppery-com/meetnav-frontend-base) before***
[![gitter](https://icon-icons.com/icons2/2530/PNG/64/gitter_button_icon_151850.png)](https://gitter.im/meetnav/community) Let's talk

Extend project [neko](https://github.com/helppery-com/neko) (submodule) adding video/audio for participants 

![image](https://user-images.githubusercontent.com/767180/108106766-ab81a780-708e-11eb-98a2-bcbb40a8e148.png)

* User will have a button to turn video on/off audio on/off
* Evaluate integrating [RTCMultiConnection](https://github.com/helppery-com/RTCMultiConnection)

We expect this functionality to be added to neko main repository, please https://n.eko.moe/#/?id=neko and follow their best practices and guidelines

> This issue will be paid by sponsors
> Please post a comment below if you want to be assigned to this issue

**Comment by shahabgohar** [comment]
@gbrian 
can you highlight pre requisite for this project ?
what we should know from technical perspective before starting this project ?

**Comment by gbrian** [comment]
Sorry, this still not clear for me yet. I'll be working on this and another issue related.
I'll keep you posted. Thanks

**Comment by gbrian** [comment]
@shahabgohar
Want to test video conference on top of neko.
For this I think the easiest way is to extend neko client with RTCMultiConnection
 
Neko has builtin support for WebRTC but is written in GO and I'd prefer to test this before going deeper.

RTCMultiConnection is quite easy to user and has good functionality, many things will be overlapping but is fine right now, it can be tested with local resources and has many examples we can take.

Can you please check the cost/viability of adding RTCMultiConnection on top of it as a prove of concept?

Goal is to have video conference on Neko, once we have a basic version working we can discuss with neko owner the best way to integrate

Epected funcionalty added to neko:
* Connect disconnect camera
* See other's with camera on
* Hear other's with audio on

Thanks

**Comment by gbrian** [comment]
discarded
